lib/cms-providers/dato.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2020 Vercel Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Job, Sponsor, Stage, Speaker } from '@lib/types';
+
+const API_URL = 'https://graphql.datocms.com/';
+const API_TOKEN = process.env.DATOCMS_READ_ONLY_API_TOKEN;
+
+async function fetchCmsAPI(query: string, { variables }: { variables?: Record<string, any> } = {}) {
+  const res = await fetch(API_URL, {
```

Comment:
I see a few of these types of fetch function scattered throughout and I'm wondering if it will be worth it to add more safety around them. For example if the `res` is not in json format or if the fetch throws an error itself.

Reply:
Not a bad idea! We will likely lean on the community for that, as the focus is on getting these CMS examples added. But I agree with you 😄